claim/gql_mutations.py

```diff
@@ -311,6 +320,11 @@ def update_or_create_claim(data, user):
     return claim
 
 
+def __autogenerate_claim_code():
+    from .utils import __autogenerate_nepali_claim_code
+    return __autogenerate_nepali_claim_code()
```

Comment:
I don't like hardcoded code generation in the "generic" code. One idea could be adding config that would point to what generator should be imported and used (example: `code_generator = 'claim.utils.nepali_claim_generator'`) this would allow us to have generic approach that is consistent with django (similar to configs in settings.py) and it wouldn't force developers to put implementation specific code in claims module (could be added as an extension).

Reply:
That was my intention, so basically, something like:
`        data['code'] = ClaimConfig.code_generator()`
should do the trick?

Reply:
That should work, please take note that apps.py is quite specific piece of code used during setup and there's a chance that code_generator will have to be lazy initialized.